ts/deployToGhPages.ts

```diff
@@ -0,0 +1,73 @@
+import { Executable } from "./executable.ts";
+import { Package } from "./package.ts";
+import { Plugin } from "./project.ts";
+
+const ansiBold = "\\e[0;1m";
+const ansiReset = "\\e[0m";
+const ansiRedBold = "\\e[31;1m";
+
+/**
+ * A garn plugin that allows easy deployment of a package to GitHub pages.
```

Comment:
Maybe link to github pages documentation here?

Reply:
I added a link to  https://pages.github.com/. Let me know if you think something like https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site would be better